src/libstrongswan/ipsec/ipsec_types.c

```diff
@@ -37,10 +37,11 @@ ENUM(ipcomp_transform_names, IPCOMP_NONE, IPCOMP_LZJH,
 	"IPCOMP_LZJH"
 );
 
-ENUM(hw_offload_names, HW_OFFLOAD_NO, HW_OFFLOAD_AUTO,
+ENUM(hw_offload_names, HW_OFFLOAD_NO, HW_OFFLOAD_FULL,
```

Comment:
YOu deleted AUTO here

Reply:
it is not deleted, still valid. it was replaced as HW_OFFLOAD_FULL become the highest value of enum.
ENUM is a nice macro to define enum list. it accepts first and last value. see bellow:

#define ENUM(name, first, last, ...) \
        ENUM_BEGIN(name, first, last, __VA_ARGS__); ENUM_END(name, last)

Reply:
got it